ledger/participant-integration-api/src/main/scala/platform/apiserver/services/transaction/ApiTransactionService.scala

```diff
@@ -139,9 +142,17 @@ private[apiserver] final class ApiTransactionService private (
       }
       .getOrElse(
         Future.failed(
-          Status.NOT_FOUND
-            .withDescription(s"invalid eventId: ${request.eventId}")
-            .asRuntimeException()
+          errorsVersionsSwitcher.choose(
+            v1 = {
+              Status.NOT_FOUND
+                .withDescription(s"invalid eventId: ${request.eventId}")
+                .asRuntimeException()
+            },
+            v2 = {
+              // TODO DPP-417, DPP-613: Use V2 error codes
+              ???
```

Comment:
Can you replace with an actual new error code? I think this [one](https://github.com/digital-asset/daml/blob/main/ledger/participant-integration-api/src/main/scala/platform/apiserver/error/LedgerApiErrors.scala#L102) would do

Reply:
Ok

FYI, I had an attempt at that for a while on this PR: https://github.com/digital-asset/daml/pull/11035/commits/60b95d96a0e2bddcda60fb4622d6a3323195cd66#diff-6822c4681530e507ebbfbdb4fd92f0a970012ff05f7ffb31084e369b869b27dcL154